docs/data/toolpad/core/introduction/integration.md

```diff
@@ -0,0 +1,757 @@
+---
+title: Toolpad Core - Integration
+description: How to integrate Toolpad Core into your existing project.
+---
+
+# Integration
+
+This guide will walk you through the process of adding Toolpad Core to an existing project.
+
+## Installation
+
+<codeblock storageKey="package-manager">
+
+```bash npm
+npm install -S @toolpad/core
+```
+
+```bash yarn
+yarn add @toolpad/core
+```
+
+```bash pnpm
+pnpm add @toolpad/core
+```
+
+</codeblock>
+
+## Next.js App Router
+
+Use the following steps to integrate Toolpad Core into your Next.js app:
+
+### 1. Wrap your application with `AppProvider`
+
+1. In your root layout file (e.g., `app/layout.tsx`), wrap your application with the `AppProvider`:
+
+```tsx title="app/layout.tsx"
+import { AppProvider } from '@toolpad/core';
+import { AppRouterCacheProvider } from '@mui/material-nextjs/v14-appRouter';
+
+export default function RootLayout({ children }: { children: React.ReactNode }) {
+  return (
+    <AppRouterCacheProvider options={{ enableCssLayer: true }}>
+      <AppProvider navigation={NAVIGATION} branding={BRANDING}>
+        {children}
+      </AppProvider>
+    </AppRouterCacheProvider>
+  );
+}
+```
+
+You can find details on the `AppProvider` props in the [base concepts](/toolpad/core/introduction/base-concepts/#app-provider) section. The [Material UI Next.js App Router integration](https://mui.com/material-ui/integrations/nextjs/) has more details on the `AppRouterCacheProvider`.
```

Comment:
Instead of linking to the base concepts page, I think we should link to the App Provider component page.